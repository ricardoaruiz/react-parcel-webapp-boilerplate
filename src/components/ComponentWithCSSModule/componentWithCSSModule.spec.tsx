import React from 'react'
import { render, screen } from '@testing-library/react'
import { ComponentWithCSSModule } from '.'

it('teste', () => {
	const { container } = render(<ComponentWithCSSModule />)
  expect(container.parentElement).toMatchSnapshot()
  
})