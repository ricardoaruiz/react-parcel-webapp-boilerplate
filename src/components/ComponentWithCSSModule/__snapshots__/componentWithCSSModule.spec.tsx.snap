// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`teste 1`] = `
<body>
  <div>
    <div>
      Component With CSS Module
    </div>
  </div>
</body>
`;
